app/helpers/users_helper.rb

```diff
@@ -1,2 +1,19 @@
 module UsersHelper
+
+  def posts_empty?(user)
+
+    unless user.posts.count > 0
+      return true
+    end
+
+  end
```

Comment:
Great :) For `user.posts.count > 0` is a boolean check, the whole method could be simplified to:

```ruby
def posts_empty?(user)
  user.posts.count == 0
end
```